qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -177,18 +304,19 @@ def __init__(self, parent: 'ZNB', name: str, channel: int,
                            set_cmd=f'SOUR{n}:POW {{:.4f}}',
                            get_parser=float,
                            vals=vals.Numbers(self._min_source_power, 25))
-        # there is an 'increased bandwidth option' (p. 4 of manual) that does
-        # not get taken into account here
         self.add_parameter(name='bandwidth',
                            label='Bandwidth',
                            unit='Hz',
                            get_cmd=f'SENS{n}:BAND?',
-                           set_cmd=f'SENS{n}:BAND {{:.4f}}',
+                           set_cmd=self._set_bandwidth,
                            get_parser=int,
                            vals=vals.Enum(
                                *np.append(10 ** 6,
                                           np.kron([1, 1.5, 2, 3, 5, 7],
-                                                  10 ** np.arange(6))))
+                                                  10 ** np.arange(6)))),
+                           docstring="There is an 'increased bandwidth' "
+                                     "option' (p. 4 of manual) that does not "
+                                     "get taken into account here."
```

Comment:
Please start the docstring with explaining what this actually is. 
Something like. 
```
The bandwidth used for sensing. Note There is an 'increased bandwidth option' (p. 4 of manual) that does not get taken into account here.
```